packages/client/hmi-client/src/components/documents/tera-document-asset.vue

```diff
@@ -2,184 +2,285 @@
+		<template #edit-buttons>
+			<SelectButton
+				:model-value="view"
+				@change="if ($event.value) view = $event.value;"
+				:options="viewOptions"
+				option-value="value"
+				option-disabled="disabled"
+			>
+				<template #option="{ option }">
+					<i
+						:class="`${
+							!pdfLink &&
+							option.value !== DocumentView.EXTRACTIONS &&
+							option.value !== DocumentView.NOT_FOUND
+								? 'pi pi-spin pi-spinner'
+								: option.icon
+						} p-button-icon-left`"
+					/>
+					<span class="p-button-label">{{ option.value }}</span>
+				</template>
+			</SelectButton>
+		</template>
+		<Accordion
+			v-if="view === DocumentView.EXTRACTIONS"
+			:multiple="true"
+			:active-index="[0, 1, 2, 3, 4, 5, 6, 7]"
+		>
+			<AccordionTab v-if="!isEmpty(formattedAbstract)">
+				<template #header>
+					<header id="Abstract">Abstract</header>
+				</template>
+				<p v-html="formattedAbstract" />
+			</AccordionTab>
+			<AccordionTab v-if="!isEmpty(figures)">
+				<template #header>
+					<header id="Figures">
+						Figures<span class="artifact-amount">({{ figures.length }})</span>
+					</header>
+				</template>
+				<ul>
+					<li v-for="(ex, index) in figures" :key="index" class="extracted-item">
+						<Image id="img" class="extracted-image" :src="ex.metadata?.url" :alt="''" preview />
+						<tera-show-more-text
+							:text="highlightSearchTerms(ex.metadata?.content ?? '')"
+							:lines="previewLineLimit"
+						/>
+					</li>
+				</ul>
+			</AccordionTab>
+			<AccordionTab v-if="!isEmpty(tables)">
+				<template #header>
+					<header id="Tables">
+						Tables<span class="artifact-amount">({{ tables.length }})</span>
+					</header>
+				</template>
+				<ul>
+					<li v-for="(ex, index) in tables" :key="index" class="extracted-item">
+						<div class="extracted-image">
+							<Image id="img" :src="ex.metadata?.url" :alt="''" preview />
+							<tera-show-more-text
+								:text="highlightSearchTerms(ex.metadata?.content ?? '')"
+								:lines="previewLineLimit"
+							/>
+						</div>
+					</li>
+				</ul>
+			</AccordionTab>
+			<AccordionTab v-if="!isEmpty(equations)">
+				<template #header>
+					<header id="Equations">
+						Equations<span class="artifact-amount">({{ equations.length }})</span>
+					</header>
+				</template>
+				<ul>
+					<li v-for="(ex, index) in equations" :key="index" class="extracted-item">
+						<div class="extracted-image">
+							<Image id="img" :src="ex.metadata?.url" :alt="''" preview />
+							<tera-show-more-text
+								:text="highlightSearchTerms(ex.metadata?.content ?? '')"
+								:lines="previewLineLimit"
+							/>
+						</div>
+						<tera-math-editor v-if="ex.metadata.equation" :latex-equation="ex.metadata.equation" />
+					</li>
+				</ul>
+			</AccordionTab>
+		</Accordion>
+		<!-- Adding this here for now...we will need a way to listen to the extraction job since this takes some time in the background when uploading a doucment-->
+		<p
+			class="pl-3"
```

Comment:
This is a no go personally.

Reply:
Sorry I made a revert as I accidentally pushed my changes for the tera-document asset rewrite here, this code isnt included